optimade_client/summary.py

```diff
@@ -172,6 +167,15 @@ class DownloadChooser(ipw.HBox):
             "Quantum ESPRESSO File [via ASE] (.in)",
             {"ext": ".in", "adapter_format": "ase", "final_format": "espresso-in"},
         ),
+        (
+            "XMol XYZ File [via ASE] (.xyz)",
+            {"ext": ".xyz", "adapter_format": "ase", "final_format": "xyz"},
+        ),
+        (
+            "Crystallographic Information File v1.0 (.cif)",
+            {"ext": ".cif", "adapter_format": "cif"},
+        ),
```

Comment:
This should be simply commented out as it can be confusing to have the same conversion option twice with a (for some users) cryptic difference of using ASE. But it's good to keep it in the code as a reference and information that it exists. So commenting these lines out should be good enough.
```suggestion
        # (
        #     "Crystallographic Information File v1.0 (.cif)",
        #     {"ext": ".cif", "adapter_format": "cif"},
        # ),
```